packages/automerge-repo-react-hooks/src/useRemoteAwareness.ts

```diff
@@ -11,17 +11,17 @@ export const peerEvents = new EventEmitter()
 
 export interface UseRemoteAwarenessProps<T> {
   /** The handle to receive ephemeral state on */
-  handle: DocHandle<T>
-  /** Our user ID */
+  handle?: DocHandle<unknown>
+  /** Our user ID - Automerge BroadcastChannel sometimes sends us our own messages; optionally filters them */
   localUserId?: string
   /** How long to wait (in ms) before marking a peer as offline */
   offlineTimeout?: number
-  /** Function to provide current epoch time */
+  /** Function to provide current epoch time (used for testing) */
```

Comment:
Probably a slightly bigger (and separate) project, but I wonder if we can use [mockdate](https://www.npmjs.com/package/mockdate) or another similar package instead to avoid this API wart. I've had a pretty good experience with mockdate in the past.